src/main/java/io/vertx/circuitbreaker/impl/CircuitBreakerImpl.java

```diff
@@ -16,16 +16,8 @@
 
 package io.vertx.circuitbreaker.impl;
 
-import io.vertx.circuitbreaker.CircuitBreaker;
-import io.vertx.circuitbreaker.CircuitBreakerOptions;
-import io.vertx.circuitbreaker.CircuitBreakerState;
-import io.vertx.circuitbreaker.OpenCircuitException;
-import io.vertx.circuitbreaker.TimeoutException;
-import io.vertx.core.Context;
-import io.vertx.core.Future;
-import io.vertx.core.Handler;
-import io.vertx.core.Promise;
-import io.vertx.core.Vertx;
+import io.vertx.circuitbreaker.*;
+import io.vertx.core.*;
```

Comment:
guessing conventionally the Vert.x project conventions allow star imports

Reply:
yes